### ISSUE TITLE
[BUG] Keyboard tab/voiceover navigation over modal does not work correctly

### ISSUE THREAD

**Issue by sorenfruergaard** [kirbydesign/designsystem#1009]
**Describe the bug**
After opening a modal and navigating using voice over through phone accessibility tools, the navigation goes over the elements in the underlying parent page, and not in the active modal.

**To Reproduce**

1. Activate voiceover/talkback
2. Open one of the modals on https://cookbook.kirby.design/examples/modal
3. Tab and listen to the underlying element being read.

**Kirby version**
- 1.2.11

**Expected behavior**
After opening a modal, the tab navigation should start inside the modal.

**Please complete the following information:**
- OS: Android
- Browser Chrome
- Browser version 81.0.4044.117

**Additional context**
Same as https://github.com/ionic-team/ionic/issues/17126

## Tasks:
- [ ] Move issue from "Ready to do" -> "in progress" on our [project board](https://github.com/kirbydesign/designsystem/projects/1)
- [ ] Create Fix Branch
- [ ] Create test reproducing the error
- [ ] Fix
- [ ] If relevant [Update Examples and Showcase](https://cookbook.kirby.design/home/showcase/button)
- [ ] Push commits to GitHub
- [ ] Open a pull request (remember to add `closes #issueno` to the description) and ask @kirbydesign/kirby-guild for review
- [ ] Merge to master and delete branch
